webpack.config.js

```diff
@@ -0,0 +1,59 @@
+const Path = require('path');
+const { JavascriptWebpackConfig, CssWebpackConfig } = require('@silverstripe/webpack-config');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+
+const PATHS = {
+  ROOT: Path.resolve(),
+  SRC: Path.resolve('client/src'),
+  DIST: Path.resolve('client/dist'),
+  MODULES: 'node_modules',
+};
+
+const config = [
+  // JavaScript
+  new JavascriptWebpackConfig('js', PATHS, 'silverstripe/htmleditor-tinymce')
+    .setEntry({
+      bundle: `${PATHS.SRC}/bundle.js`,
+      // default plugins
+      'TinyMCE_sslink': `${PATHS.SRC}/plugins/TinyMCE_sslink.js`,
+      'TinyMCE_sslink-external': `${PATHS.SRC}/plugins/TinyMCE_sslink-external.js`,
+      'TinyMCE_sslink-email': `${PATHS.SRC}/plugins/TinyMCE_sslink-email.js`,
+      // asset plugins
+      'TinyMCE_ssmedia': `${PATHS.SRC}/plugins/TinyMCE_ssmedia.js`,
+      'TinyMCE_ssembed': `${PATHS.SRC}/plugins/TinyMCE_ssembed.js`,
+      'TinyMCE_sslink-file': `${PATHS.SRC}/plugins/TinyMCE_sslink-file.js`,
+      // cms plugins
+      'TinyMCE_sslink-internal': `${PATHS.SRC}/plugins/TinyMCE_sslink-internal.js`,
+      'TinyMCE_sslink-anchor': `${PATHS.SRC}/plugins/TinyMCE_sslink-anchor.js`,
```

Comment:
Plugins need to be exported independently because of the way TinyMCE wants to pull them in.